BUILDING.md

```diff
@@ -1,11 +1,51 @@
+# Build-System CLI
+
+## Non-interactive
+```
+python build.py -h, --help
+
+usage: build.py [-h] --target {android,linux,macos,win32} --arch {x86,x64,arm}
+                [--vendor VENDOR] [--keep-native-libs] [--node-enabled]
+                [--docker] [--vagrant] [--sys-image SYS_IMAGE] [--no-shutdown]
+                [--interactive]
+                [build-steps [build-steps ...]]
+```
+```
+python build.py -v alpine -t linux -a x64 -dkr -img openjdk:8u131-alpine -ne j2v8
+```
+
+## Interactive
+```
+python build.py --i, --interactive
+
+entering interactive mode...
+
+[0] Docker >> android-x86 >> NODE_ENABLED
+[1] Docker >> android-arm >> NODE_ENABLED
+[2] Docker >> alpine-linux-x64 >> NODE_ENABLED
+[3] Docker >> linux-x64 >> NODE_ENABLED
```

Comment:
That being said, I also can imagine many ways to improve upon this feature and see what would be the best DX for the interactive CLI for devs that want to build J2V8 themselves.
But that would be stuff for a next PR ... this one is already bloated enough 😆